src/utils/auth/passport/deserialise.ts

```diff
@@ -1,14 +1,8 @@
 import { Done } from "./strategy";
-import prisma from "src/utils/prisma";
+import { User } from "@prisma/client";
 
-const deserialiseUserFunction = async (email: string, done: Done) => {
+const deserialiseUserFunction = async (user: User, done: Done) => {
   try {
-    const user = await prisma.user.findFirst({
-      where: {
-        email: email,
-      },
-    });
-
```

Comment:
It is done because in the serialise user function , I am using the entire user object in the DONE method of passport and not the email, so there is no point is seraching again by email in the deserialise user function